.github/workflows/feature.yaml

```diff
@@ -50,3 +50,13 @@ jobs:
           docker login docker.pkg.github.com -u ${GITHUB_REPOSITORY} -p ${GITHUB_TOKEN}
           docker push ${IMAGE}
           echo ${IMAGE}
+  security:
```

Comment:
Noen spesiell grunn til at det er en helt egen job, i stedet for et steg i den eksisterende jobben?

Reply:
Ja! Da kjøres den i parallel, og forstyrrer minst mulig med resten av pipeline. :D